HLTrigger/Configuration/test/OnLine_HLT_PIon.py

```diff
@@ -4001,6 +4001,7 @@
   applyAlignment = cms.bool( True ),
   alignmentsLabel = cms.string( "" )
 )
+process.TrackerAdditionalParametersPerDetESModule = cms.ESProducer("TrackerAdditionalParametersPerDetESModule")
```

Comment:
@adewit, direct modification of the HLT menu are disallowed as per TSG policy. 
Please add a customization function in [customizeHLTforCMSSW](https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/customizeHLTforCMSSW.py)

Reply:
Done